test/k8sT/DatapathConfiguration.go

```diff
@@ -209,10 +205,17 @@ var _ = Describe("K8sDatapathConfig", func() {
 			}
 
 			deploymentManager.Deploy(helpers.CiliumNamespace, IPSecSecret)
-			deploymentManager.DeployCilium(map[string]string{
+			options := map[string]string{
 				"kubeProxyReplacement": "disabled",
 				"encryption.enabled":   "true",
-			}, DeployCiliumOptionsAndDNS)
+				"tunnel":               "vxlan",
+			}
+			if helpers.RunsOnGKE() {
+				// We need to disable gke.enabled as it disables tunneling.
+				options["gke.enabled"] = "false"
+				options["endpointRoutes.enabled"] = "true"
+			}
```

Comment:
It looks like this bit is repeated across several tests. Would it make sense to move it to a helper?